dmutils/documents.py

```diff
@@ -190,6 +190,16 @@ def get_agreement_document_path(framework_slug, supplier_id, supplier_name, docu
     )
 
 
+def get_countersigned_agreement_document_path(framework_slug,
+                                              supplier_id,
+                                              document_name="countersigned-framework-agreement.pdf"):
```

Comment:
Why does `document_name` need to be parameterised? If the function is for just getting countersigned agreement document paths (as the name suggests) then the filename should always be the same shouldn't it?